setup.py

```diff
@@ -5,7 +5,7 @@
 except ImportError:
     from distutils.core import setup
 
-version = '0.6.0'
+version = '0.6.2'
```

Comment:
Whoops - Phillip should have updated it last time - my bad!

Reply:
I got your back!